tests/e2e-kubernetes/manifests.yaml

```diff
@@ -5,7 +5,7 @@ ShortName: ebs
 StorageClass:
   FromFile: storageclass.yaml
 SnapshotClass:
-  FromFile: snapshotclass.yaml
+  FromName: true
```

Comment:
Not sure I agree with this, would love to see other's thoughts

Reply:
`FromFile` would be more appropriate if we had additional parameters in the snapshotclass object. One benefit of this change is that we no longer have to maintain our own snapshotclass for testing.

https://github.com/kubernetes/kubernetes/blob/5556d3336b986dcd7ec7de85f731425be91273b4/test/e2e/storage/external/external.go#L99

Reply:
We've had issues using this one because it _copies_ the given object per test and when the original object is also the default, you end up with multiple defaults resulting in failures. I tried to fix this in k/k for StorageClass, would assume its the same for SnapshotClass. https://github.com/kubernetes/kubernetes/pull/111730

Reply:
@olemarkus [FromExistingClassName](https://github.com/kubernetes/kubernetes/blob/5556d3336b986dcd7ec7de85f731425be91273b4/test/e2e/storage/external/external.go#L103) behaves as you describe, but we specify [FromName]() here which I believe behaves differently. Either way, I can revert this change if you / other reviewers feel strongly about it.

Reply:
In that case, I think it's fine as-is